app/views/home/show.html.slim

```diff
@@ -1,6 +1,17 @@
 = content_for :sidebar
   = render 'home/sidebar'
 
+- if session[:new_organization]
```

Comment:
@dissident I think you may use `flash` instead. It already has similar functionality